src/components/homepage/homepage.component.html

```diff
@@ -1 +1,41 @@
+<div class="hero-banner__container">
+	<div class="hero-banner__text">
+		<div class="hero-banner__text-heading">Be The Reason</div>
+		<div class="hero-banner__text-heading">Someone Smiles Today</div>
+		<div class="hero-banner__text-description">
+			Our platform bridges the gap between those with an abundance and those in
+			need, fostering a community of kindness and generosity. Together, we turn
+			surplus into sustenance, transforming every meal into a beacon of hope and
+			compassion.
+		</div>
+		<div class="hero-banner__text-description">
+			Join us in spreading love, one plate at a time !
 
+		</div>
+		<div class="hero-banner__text-buttons">
+			<button type="button" class="hero-banner__text-buttons-donate btn">
+				Donate
+				<img
+					class="hero-banner__text-buttons-donate-icon"
+					alt="donate"
+					src="../../assets/icons/homepage/gift.svg"
+				/>
+			</button>
+			<button type="button" class="hero-banner__text-buttons-grab btn">
+				Grab a Dish
```

Comment:
Rather than "Grab a dish"  it should be something like "Receive Food" or something like it. Dish should not be there